src/helpers/on_fail/index.js

```diff
@@ -1,19 +1,24 @@
 // @flow
 const WARN: string = 'warn';
 
+type CountName = 'failCount' | 'warnCount';
```

Comment:
I agree with you, but since it is a breaking change I intend to handle it in a a future PR.
- add depreciation warnings for `fail`, start accepting `error` as well. Deprecate fail in the next major.